app/packages/registration-schema/validations.ts

```diff
@@ -98,8 +98,8 @@ export function validateNodeLicense() {
 }
 
 export function validateSubjects() {
-    return async (_: unknown, __: unknown, ___: unknown, ____: unknown, content: DraftRegistration) => {
-        const subjects = await content.subjects;
+    return (_: unknown, __: unknown, ___: unknown, ____: unknown, content: DraftRegistration) => {
+        const subjects = content.hasMany('subjects').value();
```

Comment:
we don't fetch the subjects in the validation function anymore. Instead, we look at what's already in the store.